app/1.0/docs/tools/polymer-cli.md

```diff
@@ -275,19 +275,44 @@ Open up a page other than the default `index.html` in a specific browser
 
     polymer serve --open app.html --browser Safari
 
-### Lint element(s) {#lint}
+### Lint {#lint}
 
-Check elements in your element project or app project for syntax errors, anti-patterns and more.
+Analyze your project for syntax errors, missing imports, bad databinding expressions and more.
 
-Element project example:
+Use it like so:
 
-    polymer lint --input my-el.html
+    polymer lint --rules=polymer-1
 
-App project example (linting multiple elements):
+This will lint all of the code in your project with the `polymer-1` ruleset, which is appropriate for projects using Polymer 1.0. If you're upgrading to Polymer 2.0, then the `polymer-2-hybrid` or `polymer-2` are a better choice, as they'll warn you about use of deprecated Polymer 1.0 features.
 
-    polymer lint --root src/ --input my-app/my-app.html my-el/my-el.html
+You can pass flags to the linter like `--rules` but even better is to put the configuration in `polymer.json` so that all you need to do is run `polymer lint`. Putting your configuration in `polymer.json` also means that other tools, like IDE plugins can use the same lint configuration.
 
-If all of the elements you want to test are in the same directory, you can specify the `--root` flag to make all of the `--input` files relative to that directory.
+Here's what that looks like:
+
+```json
+{
+  "lint": {
+      "rules": ["polymer-1"],
+      "ignoreWarnings": []
+  }
+}
+```
+
+- `rules`: An array of lint rules and rule collections to run on your project. For most projects, one of `polymer-1`, `polymer-2-hybrid`, or `polymer-2` is all that's needed here.
+- `ignoreWarnings`: An array of warning codes to ignore.
+
+#### Warning Codes:
+
+The output of `polymer lint` looks like this:
+
+```
+            <iron-collapse>
+            ~~~~~~~~~~~~~~~
+
+index.html(83,12) warning [undefined-elements] - The element iron-collapse is not defined
+```
+
+This means that on line 83 of `index.html` there's an `<iron-collapse>` tag, but the linter can't find the definition of the `iron-collapse` custom element. This probably means that there's a missing HTML import in `index.html`, but if we wanted to ignore this warning, we'd add `"undefined-elements"` to the `"ignoreWarnings"` array in `polymer.json`.
```

Comment:
Suggest breaking the last sentence and writing in 2nd person for consistency:

in `index.html`. To ignore this warning, add `"undefined-elements"` to the `"ignoreWarnings"` array in `polymer.json`.